src/abstractdataframe/abstractdataframe.jl

```diff
@@ -2588,7 +2588,10 @@ function _compile_permutation!(p::AbstractVector{<:Integer})
     return resize!(out, out_len)
 end
 
-# permute a vector `v` based on a permutation `p` listed in cycle notation
+# Permute a vector `v` based on a permutation `p` listed in zero terminated
+# cycle notation. For example, the permutation 1 -> 2, 2 -> 3, 3 -> 1, 4 -> 6,
+# 5 -> 5, 6 -> 4 is traditionaly expressed as [2, 3, 1, 6, 5, 4] but in cycle
+# notation is expressed as [1, 2, 3, 0, 4, 6, 0] (or [1, 2, 3, 0, 4, 6, 0, 5, 0])
```

Comment:
```suggestion
# notation used in the implementation is expressed as [1, 2, 3, 0, 4, 6, 0]
```
I would remove the alternative as we do not use it

Reply:
okay